airflow/operators/bash_operator.py

```diff
@@ -74,7 +74,7 @@ def execute(self, context):
                 f.write(bytes(bash_command, 'utf_8'))
                 f.flush()
                 fname = f.name
-                script_location = tmp_dir + "/" + fname
+                script_location = fname
```

Comment:
maybe use `os.path.join` instead? Having the name of the temporary folder in the log can be useful for debugging

Reply:
f.name returns the full path to the temp file (e.g. https://docs.python.org/3/library/tempfile.html#deprecated-functions-and-variables)

The script_location variable is redundant here and should be removed

Reply:
`
>>> with TemporaryDirectory(prefix='airflowtmp') as tmp_dir:
...   with NamedTemporaryFile(dir=tmp_dir, prefix="my_prefix") as f:
...     print f.name
...
/var/folders/5y/z0j_5sn53_j1rhh_zhlhr8x80000gn/T/airflowtmpuc98p4/my_prefixEDzoYK
`